hoomd/md/PotentialPair.h

```diff
@@ -178,6 +178,51 @@ template<class evaluator> class PotentialPair : public ForceCompute
         m_attached = false;
         }
 
+    //! Get the contribution to the external virial
+    Scalar getExternalVirial(unsigned int dir)
+        {
+        assert(dir < 6);
+
+        return m_external_virial[dir];
```

Comment:
Return 1/3 of \delta W in elements 0, 3, and 5 (return 0 for other elements). \delta W = \delta P * D * V - where D is the dimensionality of the system (2 or 3) and V is the box volume (or area).